[report]
fail_under = 92

[run]
source = pystac
